docs/001_develop/03_client-capabilities/010_filters/index.mdx

```diff
@@ -111,28 +111,32 @@ These flavours are explained in detail below with help of code examples.
-* [urlTargeting](./docs/api/foundation-filters.urltargetingfilter.md): It is used to filter content based on URL parts like scheme, path, host, or pattern matching. It allows to target specific URLs based on particular criteria, which can be useful for enabling or restricting features.
-* [userAgent](./docs/api/foundation-filters.useragentfilter.md): It filters content based on user's browser, OS, devices or specified user agent string. It allows customized experience according to specific environments, such as targeting mobile devices audience or restricting access on a particular browser.
-* [userTargeting](./docs/api/foundation-filters.usertargetingfilter.md): This filters content on basis of user related attributes like usernames, permissions and profiles. It aids in showing personalized content or restricting access to specific content depending on the soecified user conditions.
+* [nodeEnv](./docs/api/foundation-filters.nodeenvfilter.md): This filter is used to match the current environment (production, development etc.) with the environment specified ( which is passed as function param). It is useful for supporting environment-specific conditions so that certain features are only available for targeted environment.
+* [percentage](./docs/api/foundation-filters.percentagefilter.md): The percentage filter checks if a randomly generated number between 0 and 100 is within a specified percentage threshold. This enables you to set probabilistic conditions, so that features or elements can be displayed to a proportion of users based on the given percentage. This filter is useful for scenarios such as A/B testing or gradually rolling out new features.
+* [timeWindow](./docs/api/foundation-filters.timewindowfilter.md): This filter checks whether the current date and time come fall within the start and end period specified in the parameters. The filter returns true if the current date and time falls between the range, thus making it useful for time-sensitive conditions.
+* [urlTargeting](./docs/api/foundation-filters.urltargetingfilter.md): This filters content by URL parts, such as scheme, path, host, or pattern matching. It enables you to target specific URLs, which can be useful for enabling or restricting features.
+* [userAgent](./docs/api/foundation-filters.useragentfilter.md): This filters content by the user's browser, OS, devices or specified user agent string. For example, you can customize the experience according to specific environments, such as targeting mobile devices or audience, or you can restrict access from a specific browser.
+* [userTargeting](./docs/api/foundation-filters.usertargetingfilter.md): This filters content on basis of attributes such as usernames, permissions and profiles. It enables you to show personalized content or to restrict access to specific content depending on user conditions.
 
 
 ## Installation
```

Comment:
Nothing to do with your change but while we're at it - since the docs rejuvination we've tended to favour having the installation instructions at the bottom, as they're only something you do once, and then you can link to it from here. I think we might want to do that here too